src/test/java/org/reaktivity/specification/nukleus/http_cache/streams/Rfc7234ProxyConnectIT.java

```diff
@@ -428,4 +428,28 @@ public void shouldProxyResponseTooLargeToCache() throws Exception
         k3po.notifyBarrier("ROUTED_PROXY");
         k3po.finish();
     }
+
+    @Test
+    @Specification({
+            "${streams}/invalidate.multiple.max-age/connect/client",
+            "${streams}/invalidate.multiple.max-age/connect/server",
+    })
+    public void shouldNotCacheWithMultipleMaxAge() throws Exception
```

Comment:
I think you are missing test methods here

Reply:
What do mean?